src/test/java/com/nvidia/spark/rapids/jni/CastStringsTest.java

```diff
@@ -59,6 +59,60 @@ void castToIntegerTest() {
     }
   }
 
+
+  //This is for testing the CastStrings class with the input of "inf" etc string
```

Comment:
nit: This testcase looks self-describing, how about removing the comments? Also for `castToFloatNanTest`.

Reply:
Done